compiler/main.bal

```diff
@@ -25,18 +26,18 @@ public function main(string[] filenames, *Options opts) returns error? {
             check showTypes(filename);
             continue;
         }
-        check compileFile(filename, check chooseOutputFilename(filename, opts.outDir));
+        check compileFile(filename, check chooseOutputFilename(filename, opts.outDir), {gcName: opts.gc});
     }  
 }
 
 //  outputFilename of () means don't output anything
-function compileFile(string filename, string? outputFilename) returns CompileError {
+function compileFile(string filename, string? outputFilename, *nback:Options options) returns CompileError {
```

Comment:
Use a name for `options` that makes it clear that it is nback options you are talking about e.g. `nbackOptions`

Reply:
Fixed